storage/storage.cpp

```diff
@@ -105,6 +106,36 @@ void Storage::Clear()
   m_localFilesForFakeCountries.clear();
 }
 
+bool Storage::HaveDownloadedCountries()
+{
+  return !m_localFiles.empty();
+}
+
+void Storage::Migrate()
+{
+  platform::migrate::SetMigrationFlag();
+  
+  Clear();
+  m_countries.Clear();
+  
+  LoadCountriesFile(true /* forceReload */);
+}
+
+void Storage::DeleteAllLocalMaps(vector<TIndex> * existedCountries /* = nullptr */)
+{
+  for (auto const & localFiles : m_localFiles)
+  {
+    for (auto const & localFile : localFiles.second)
+    {
+      LOG_SHORT(LINFO, ("Remove:", localFiles.first, DebugPrint(*localFile)));
+      if (existedCountries)
+        existedCountries->push_back(localFiles.first);
```

Comment:
Is it ok that localFiles.first is put into existedCountries multiple times? (actually, loalFiles.second.size() times.)